src/main/com/example009/radek/v3/FileService.java

```diff
@@ -0,0 +1,144 @@
+package main.com.example009.radek.v3;
+
+import java.io.*;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileService {
+
+    private final String PATH_FILE_RESULTS = "src/main/resources/example009/v3/";
```

Comment:
To může být static ... tak jak napovídá IDEa ... krom těch system.out.println bys mel opravit vse, co ti napovida